cmd/virt-launcher/virt-launcher.go

```diff
@@ -350,16 +335,15 @@ func main() {
 	allowEmulation := pflag.Bool("allow-emulation", false, "Allow use of software emulation as fallback")
 	runWithNonRoot := pflag.Bool("run-as-nonroot", false, "Run libvirtd with the 'virt' user")
 	hookSidecars := pflag.Uint("hook-sidecars", 0, "Number of requested hook sidecars, virt-launcher will wait for all of them to become available")
-	noFork := pflag.Bool("no-fork", false, "Fork and let virt-launcher watch itself to react to crashes if set to false")
 	ovmfPath := pflag.String("ovmf-path", "/usr/share/OVMF", "The directory that contains the EFI roms (like OVMF_CODE.fd)")
 	qemuAgentSysInterval := pflag.Duration("qemu-agent-sys-interval", 120*time.Second, "Interval between consecutive qemu agent calls for sys commands")
 	qemuAgentFileInterval := pflag.Duration("qemu-agent-file-interval", 300*time.Second, "Interval between consecutive qemu agent calls for file command")
 	qemuAgentUserInterval := pflag.Duration("qemu-agent-user-interval", 10*time.Second, "Interval between consecutive qemu agent calls for user command")
 	qemuAgentVersionInterval := pflag.Duration("qemu-agent-version-interval", 300*time.Second, "Interval between consecutive qemu agent calls for version command")
 	qemuAgentFSFreezeStatusInterval := pflag.Duration("qemu-fsfreeze-status-interval", 5*time.Second, "Interval between consecutive qemu agent calls for fsfreeze status command")
-	keepAfterFailure := pflag.Bool("keep-after-failure", false, "virt-launcher will be kept alive after failure for debugging if set to true")
 	simulateCrash := pflag.Bool("simulate-crash", false, "Causes virt-launcher to immediately crash. This is used by functional tests to simulate crash loop scenarios.")
 	libvirtLogFilters := pflag.String("libvirt-log-filters", "", "Set custom log filters for libvirt")
+	pflag.Bool("keep-after-failure", false, "virt-launcher will be kept alive after failure for debugging if set to true")
```

Comment:
Here we don't need it, but I don't want to enable the `ParseUnknowFlag`, so here I'm just saying that it can be accepted. Once thing I can do is to parse all the flags at `virt-launcher-monitor` level and remove it is present.
The rationale behind this was that I want to delegate all the flag parsing to the real `virt-launcher` and not to `virt-launcher-monitor`, but yes this questionable. Any suggests? Thanks

Reply:
The preference would be to handle this flag in one place, if possible, instead of maintaining it in both places.
I'll be fine with a follow-up PR if it's tricky to do it in this PR.
I guess we can just remove this flag from a copy of os.Args?